classes/extension/aggregator.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace mageekguy\atoum\extension;
+
+
+use mageekguy\atoum\exceptions\logic\invalidArgument;
+
+class aggregator extends \splObjectStorage
+{
+    public function getHash($object)
+    {
+        if (is_object($object) === false)
+        {
+            throw new invalidArgument(__METHOD__ . ' expects parameter 1 to be object, ' . gettype($object) . ' given');
```

Comment:
`getHash` already expect an object as its unique argument (http://php.net/manual/en/splobjectstorage.gethash.php). Is it really necessary to re-check it?